app/src/main/java/com/isc/hermes/controller/CurrentLocationController.java

```diff
@@ -0,0 +1,122 @@
+package com.isc.hermes.controller;
+
+import android.annotation.SuppressLint;
+import android.widget.ImageButton;
+import android.widget.Toast;
+
+import androidx.appcompat.app.AppCompatActivity;
+
```

Comment:
Delete the spaces that you do not get to use.

Reply:
@VillalobosGarciaGaby  This recommendation is already satisfied.